From 6e88cbad1b01fba9f849dfeea0d88d2a49d6c255 Mon Sep 17 00:00:00 2001
From: Peter Jones <pjones@redhat.com>
Date: Mon, 31 Mar 2014 15:43:10 -0400
Subject: [PATCH 15/41] Previous ARM 64-bit test had the wrong compiler define
 for aarch64.

Yet one more reason that arch name is an incredibly stupid idea.

Signed-off-by: Peter Jones <pjones@redhat.com>
---
 grubby.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/grubby.c b/grubby.c
index 3cded58..3485d4e 100644
--- a/grubby.c
+++ b/grubby.c
@@ -60,7 +60,7 @@ int debug = 0;	/* Currently just for template debugging */
 
 int isEfi = 0;
 
-#if defined(__arch64__)
+#if defined(__aarch64__)
 #define isEfiOnly	1
 #else
 #define isEfiOnly	0
-- 
2.4.3

